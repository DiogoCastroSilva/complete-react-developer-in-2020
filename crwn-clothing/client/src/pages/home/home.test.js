import React from 'react';
import { shallow } from 'enzyme';

import Home from './home.component';


describe('testing Home component', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<Home />);
    });

    it('should render a Home component', () => {
        expect(wrapper).toMatchSnapshot();
    });
});