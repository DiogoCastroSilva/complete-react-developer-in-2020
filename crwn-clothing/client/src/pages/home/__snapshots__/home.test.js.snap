// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Home component should render a Home component 1`] = `
<styled.div>
  <Connect(DirectoryContainer) />
</styled.div>
`;
